litgpt/config.py

```diff
@@ -1563,6 +1563,37 @@ def norm_class(self) -> Type:
         configs.append(copy)
 
 
+############
+# MicroLlama
+############
+micro_llama = [
+    dict(
+        name="micro-llama-300M{}",
+        hf_config=dict(org="keeeeenw", name="MicroLlama{}"),
+        block_size=2048,
+        vocab_size=32000,
+        padding_multiple=64,
+        n_layer=12,
+        n_head=16,
+        n_embd=1024,
+        rotary_percentage=1.0,
+        parallel_residual=False,
+        bias=False,
+        norm_class_name="RMSNorm",  # original TinyLlama and MicroLlama uses FusedRMSNorm
+        norm_eps=1e-5,
+        mlp_class_name="LLaMAMLP",
+        intermediate_size=5632,
+        n_query_groups=4,
+    )
+]
+for c in micro_llama:
+    for kind, hf_postfix in [("", "")]:
```

Comment:
If there is no an additional version (e.g. instruct, chat), then this for loop is not needed.
Just need to extend configs list.

Reply:
I planned to add more variants so I let it there. I can remove it for now and the code back when the variants are ready.